assets/utils/commands/bump.js

```diff
@@ -0,0 +1,163 @@
+const output = require('../utility/output');
+const Input = require('../utility/input');
+const repository = require('../utility/repository');
+const { execSync } = require('child_process');
+const path = require('path');
+const process = require('process');
+const readline = require('readline');
+
+const c = output.color;
+
+async function bump(...inputList) {
+  const autoYes = inputList.includes('--yes') || inputList.includes('-y');
+  inputList = inputList.filter(arg => arg !== '--yes' && arg !== '-y');
+
+  const input = new Input(inputList);
+  await input.init(true);
+
+  const lib = detectLibrary(input);
+  if (!lib) return;
+
+  if (!runH5pBump(lib)) return;
+
+  process.chdir(lib);
+  output.printLn(`${c.blue}Changed directory back to: ${c.emphasize}${lib}${c.default}`);
+
+  const version = getVersion();
+  if (!version) return;
+
+  if (!stageChanges(autoYes)) return;
+  if (!commitChanges(version)) return;
+
+  if (autoYes) {
+    tagAndPush(version, true, true);
+  } else {
+    promptTagAndPush(version);
+  }
+}
+
+// Helpers
+function detectLibrary(input) {
+  let libraries = input.getLibraries();
+  if (!libraries.length) {
+    const cwd = process.cwd();
+    const libName = path.basename(cwd);
+    libraries = [libName];
+    process.chdir(path.dirname(cwd));
+    output.printLn(`${c.blue}Defaulting to current folder: ${c.emphasize}${libName}${c.default}`);
+  }
+  output.printLn(`${c.blue}Bumping patch version of: ${c.emphasize}${libraries[0]}${c.default}`);
+  return libraries[0];
+}
+
+function runH5pBump(lib) {
+  let bumpOutput = '';
+  try {
+    bumpOutput = execSync(`h5p utils increase-patch-version ${lib}`, { encoding: 'utf8' });
+    output.printLn(bumpOutput.trim());
+  } catch {
+    output.printLn(`${c.red}Failed to bump version using h5p utils.${c.default}`);
+    return false;
+  }
+  if (bumpOutput.includes('SKIPPED')) {
+    output.printLn(`${c.yellow}Nothing to bump — skipping further steps.${c.default}`);
+    return false;
+  }
+  return true;
+}
+
+function getVersion() {
+  const libData = repository.getLibraryData('.');
+  const { majorVersion, minorVersion, patchVersion } = libData;
+  if (!majorVersion || !minorVersion || !patchVersion) {
```

Comment:
The version numbers (are of type `number`, not `string` aren't they?) could be `0` and `!0` is truthy.